code/lib/types/src/modules/csf.ts

```diff
@@ -58,7 +162,7 @@ export type {
   StrictGlobalTypes,
   StrictInputType,
   Tag,
-} from '@storybook/csf';
+};
```

Comment:
Keep the `from` clause of the first export block as is and simply move your export to its own block.

```suggestion
} from '@storybook/csf';

export type { InputTypeOverride as InputType };
```